Microsoft.Toolkit.Uwp.SampleApp/Pages/About.xaml

```diff
@@ -51,7 +51,10 @@
                              NavigateUri="{x:Bind Url}"
                              Style="{StaticResource AboutHyperlinkButtonStyle}">
                 <Grid>
-                    <TextBlock Text="{x:Bind FullName}" />
+                    <TextBlock Width="150"
+                               HorizontalAlignment="Left"
+                               TextTrimming="CharacterEllipsis"
+                               Text="{x:Bind FullName}" />
```

Comment:
We could also add a Tooltip here so when you hover it still has the full text easily accessible, eh?

Reply:
Like the idea, will do!

Reply:
Tooltip added.